frontend/containers/forms/VolunteerForm.js

```diff
@@ -44,16 +46,16 @@ class VolunteerForm extends Component {
   render() {
     return (
       <div>
-
-      <Col md={6} mdOffset={3} style={{backgroundColor: "rgba(255, 255, 255, 1)", paddingTop:"30px"}}>
-          <Image style={{paddingLeft: "33%"}} src={Logo} />
-          <p>Thank you for your interest in volunteering with us! You can volunteer to help us with a specific project, event, going to the homeless shelters with us or helping us out around the office. Whatever it is, you are guaranteed to leave with a full heart and ear to ear smile!</p>
-          <p>While we greatly need and appreciate all of the volunteer assistance we receive, we do not have a full time volunteer manager on staff. Thank you in advance for understanding that your application may take a few weeks to get processed.</p>
+        <Col md={6} mdOffset={3} style={{backgroundColor: "rgba(255, 255, 255, 1)", paddingTop:"30px"}}>
+            <Image style={{paddingLeft: "33%"}} src={Logo} />
+            <p>Thank you for your interest in volunteering with us! You can volunteer to help us with a specific project, event, going to the homeless shelters with us or helping us out around the office. Whatever it is, you are guaranteed to leave with a full heart and ear to ear smile!</p>
+            <p>While we greatly need and appreciate all of the volunteer assistance we receive, we do not have a full time volunteer manager on staff. Thank you in advance for understanding that your application may take a few weeks to get processed.</p>
+        
         <Form model="forms.user" onSubmit={v => console.log(v)}>
 
-        <Tabs>
-        <Tabs.Panel title='Personal Information'>
-              <h2>Personal Information</h2>
+        {/* <Tabs> */}
```

Comment:
Let's remove these Tabs elements entirely, looks like we won't be needing them in the future.